### PR TITLE
Fix for dropdown-right alignment being incorrect

### PR DESCRIPTION
**Pull request by chief10**
Changed display property for class .dropdown-menu-right to fix this issue: https://github.com/twbs/bootstrap/issues/18852

I opted to change the display property for dropdown-menu-right instead of dropdown-menu, because this issue is specific to the dropdown-menu-right class. Changing the display property for dropdown-menu class could have unforeseen consequences.,

### CODE REVIEW

File: scss/_dropdown.scss [hunk 1]
```diff
@@ -131,6 +131,7 @@
 .dropdown-menu-right {
   right: 0;
   left: auto; // Reset the default from `.dropdown-menu`
+  display: inline-block; //Make sure width doesnt stretch to fill container.
```
Comment (cvrebert) [inline scss/_dropdown.scss]: Missing an apostrophe in "doesn't"
Missing a space after the `//`

**Comment by cvrebert** [comment]
> Changing the display property for dropdown-menu class could have unforeseen consequences.

I would suspect that changing it for `.dropdown-menu-right` would have similar unforeseen consequences, so I'm doubtful about this being the right fix.

**Comment by chief10** [comment]
I would tend to agree with you. I guess the only alternative then would be to wrap the entire `.dropdown` element in something that has a set width on it -- or to give the dropdown a set width itself on a case-by-case basis -- and go from there. Unless there is some other way to do it, do you think this should be added into the documentation?

**Comment by cvrebert** [comment]
I'm hoping @mdo can apply his CSS-fu and formulate a good fix.

**Comment by mdo** [comment]
Why right align a dropdown menu when the parent button is left aligned in a full-width container? That utility class is designed for right aligned buttons.

**Comment by chief10** [comment]
I cant say for certain what the original use case was. In any case, if you say that utility class is meant for right-aligned buttons, then maybe this is more of an issue with documentation than anything else. 

Closing this one out because of that.

**Pull request closed without merging**